miniapps/meshing/pminimal-surface.cpp

```diff
@@ -0,0 +1,1302 @@
+      if (!opt.id)
+      {
+         mfem::out << "Iteration " << i << ", norm: " << norm
+                   << ", area: " << area << std::endl;
+      }
+      if (opt.vis) { Surface::Visualize(opt, &pmesh, &u); }
+      if (opt.print) { Surface::Print(opt, &pmesh, &u); }
+      if (norm < NRM) { break; }
+   }
+   return 0;
+}
+
+int main(int argc, char *argv[])
+{
+   Opt opt;
+   MPI_Init(&argc, &argv);
+   MPI_Comm_rank(MPI_COMM_WORLD, &opt.id);
+   MPI_Comm_size(MPI_COMM_WORLD, &opt.sz);
+   // Parse command-line options.
+   OptionsParser args(argc, argv);
+   args.AddOption(&opt.pb, "-p", "--problem", "Problem to solve.");
+   args.AddOption(&opt.mesh_file, "-m", "--mesh", "Mesh file to use.");
+   args.AddOption(&opt.wait, "-w", "--wait", "-no-w", "--no-wait",
+                  "Enable or disable a GLVis pause.");
+   args.AddOption(&opt.radial, "-rad", "--radial", "-no-rad", "--no-radial",
+                  "Enable or disable radial constraints in solver.");
+   args.AddOption(&opt.nx, "-x", "--num-elements-x",
+                  "Number of elements in x-direction.");
+   args.AddOption(&opt.ny, "-y", "--num-elements-y",
+                  "Number of elements in y-direction.");
+   args.AddOption(&opt.order, "-o", "--order", "Finite element order.");
+   args.AddOption(&opt.refine, "-r", "--ref-levels", "Refinement");
+   args.AddOption(&opt.niters, "-n", "--niter-max", "Max number of iterations");
+   args.AddOption(&opt.surface, "-s", "--surface", "Choice of the surface.");
+   args.AddOption(&opt.pa, "-pa", "--partial-assembly", "-no-pa",
+                  "--no-partial-assembly", "Enable Partial Assembly.");
+   args.AddOption(&opt.lambda, "-l", "--lambda", "Lambda step toward solution.");
+   args.AddOption(&opt.amr, "-a", "--amr", "-no-a", "--no-amr", "Enable AMR.");
+   args.AddOption(&opt.amr_threshold, "-at", "--amr-threshold", "AMR threshold.");
+   args.AddOption(&opt.device_config, "-d", "--device",
+                  "Device configuration string, see Device::Configure().");
+   args.AddOption(&opt.keys, "-k", "--keys", "GLVis configuration keys.");
+   args.AddOption(&opt.vis, "-vis", "--visualization", "-no-vis",
+                  "--no-visualization", "Enable or disable visualization.");
+   args.AddOption(&opt.vis_mesh, "-vm", "--vis-mesh", "-no-vm",
+                  "--no-vis-mesh", "Enable or disable mesh visualization.");
+   args.AddOption(&opt.by_vdim, "-c", "--solve-byvdim",
+                  "-no-c", "--solve-bynodes",
+                  "Enable or disable the 'ByVdim' solver");
+   args.AddOption(&opt.print, "-print", "--print", "-no-print", "--no-print",
+                  "Enable or disable result output (files in mfem format).");
+   args.Parse();
+   if (!args.Good()) { args.PrintUsage(mfem::out); MPI_Finalize(); return 1; }
+   MFEM_VERIFY(opt.lambda >= 0.0 && opt.lambda <= 1.0,"");
+   MFEM_VERIFY(!opt.vis_mesh, "Option not available in parallel!");
```

Comment:
If this option is not available then why offer it?

Reply:
You're right - `removed`